.github/workflows/git-tag-github-release.yml

```diff
@@ -0,0 +1,84 @@
+name: GitHub Release and git tag
```

Comment:
GitHub Release and tag test:
https://github.com/NexusMutual/test-ci-1/releases/tag/v3.4.1